app/lib/common/widgets/html_editor/components/mention_menu.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:acter/common/widgets/html_editor/components/mention_handler.dart';
+import 'package:acter/common/widgets/html_editor/models/mention_type.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+
+class MentionMenu {
+  MentionMenu({
+    required this.context,
+    required this.editorState,
+    required this.roomId,
+    required this.style,
+    required this.mentionType,
+  });
+
+  final BuildContext context;
+  final EditorState editorState;
+  final String roomId;
+  final MentionMenuStyle style;
+  final MentionType mentionType;
+
+  OverlayEntry? _menuEntry;
+  bool selectionChangedByMenu = false;
+
+  void dismiss() {
+    if (_menuEntry != null) {
+      editorState.service.keyboardService?.enable();
+      editorState.service.scrollService?.enable();
+      keepEditorFocusNotifier.decrease();
+    }
+
+    _menuEntry?.remove();
+    _menuEntry = null;
+  }
+
+  void _onSelectionUpdate() => selectionChangedByMenu = true;
```

Comment:
Why exactly `selectionChangedByMenu` variable will do?
I don't see usage of this here apart from setting it's value from `false` to `true`.